pages/linux/ntpd.md

```diff
@@ -0,0 +1,8 @@
+# ntpd
+
+> The official Network Time Protocol daemon program.
+> More information: <https://www.ntp.org/documentation/4.2.8-series/ntpd/>.
+
+- Synchronize system time with remote servers, allowing "Big" adjustments with --panicgate:
+
+`sudo ntpd --panicgate --quit`
```

Comment:
```suggestion
- Synchronize system time with remote servers a single time (quit after synchronizing):

`sudo ntpd --quit`

- Synchronize a single time allowing "Big" adjustments:

`sudo ntpd --panicgate --quit`
```
Distributing the options allows an easier comprehension and more concise descriptions, that's what most tldr pages do.